docs/using.md

```diff
@@ -107,11 +107,58 @@ steps:
         value: "world"
 ```
 
+### Resource shared between tasks
+
 Pipeline Tasks are allowed to pass resources from previous tasks via
 `providedBy` field. This feature is implemented using Persistent Volume Claim
 under the hood but however has an implication that tasks cannot have any volume
 mounted under path `/pvc`.
 
+### Outputs
+
+`Task` definition can include inputs and outputs resource declaration. If specific set of resources are only declared in output then a copy of resource to be uploaded or shared for next task is expected to be present under the path `/workspace/output/resource_name/`. 
+
+```yaml
+resources:
+  outputs:
+    name: storage-gcs
+steps:
+  - image: objectuser/run-java-jar #https://hub.docker.com/r/objectuser/run-java-jar/
+    command: [jar]
+    args: ["-cvf", "-o", "/workspace/output/storage-gcs/", "projectname.war", "*"]
+    env:
+    - name: "FOO"
+      value: "world"
+```
+
+**Note**: If task is relying on output resource functionality then they cannot mount anything in file path `/workspace/output`.
```

Comment:
why do we change the path when a resource is only an `output`? i'm wondering why we decided not to use the same path in both cases?

Reply:
We could use the same path in both cases, @pivotal-nader-ziada had similar suggestion too. If we take that route then it becomes more burden for operator to have a copy of resource present in path `/workspace/output`. If resource is already declared in input then we know where the resource is already placed, then we are re-using that information.

Reply:
> If we take that route then it becomes more burden for operator to have a copy of resource present in path /workspace/output.

Is the idea that inputs are always in `/workspace/` and outputs are always in `/workspace/outputs`?

An alternative could be to have unique names for all inputs + outputs, and put all of them in `/workspace/`

Reply:
True. We could do this.